### DISCUSSION TITLE
Total Points

### DISCUSSION THREAD

**Discussion by MrDGer** [cwendt94/espn-api#368] [Q&A] [answered]
Having issues with using the total_points feature, as it keeps returning zero.  Same is true for the projected_total_points.

Is box_scores[0].home_lineup[0].total_points an appropriate way to call for it? Or am I just attempting to access it in the wrong way?

**Comment by cwendt94** [comment, marked as answer]
box score will have the projected and points for that weeks matchup but the total points can be access through the teams field in the league. So `league.teams[0].roster[0].total_points`

**Reply by MrDGer** [reply]
I figured it was something on my end! Once again, thanks for all the work put into this.  Our family league now has a custom live scoreboard overlay on the games every week, and we are loving it!

**Reply by cwendt94** [reply]
No problem! Thats awesome, glad the package has been helpful!!
